src/semantics/ZPat.re

```diff
@@ -68,7 +68,7 @@ and erase' = (zp': t'): UHPat.t' =>
     OpSeq(skel, OperatorSeq.opseq_of_exp_and_surround(p1, surround));
   };
 
-let is_after = (zp: t): bool =>
+let _is_after = (zp: t): bool =>
```

Comment:
Because this isn't used anywhere I have to put an underscore to please the linter. If someone eventually needs to use it we can just delete the _.

Reply:
it complains even about top-level definitions that are unused? do we need to make .rei files to specify a public interface?

Reply:
Yeah a .rei may resolve this warning

Reply:
Turns out this definition was being shadowed! There was a newer definition at the bottom of the file (I checked the timestamps), so given that this version wasn't used I deleted it.